proto/grpc/health/v1/health.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright 2015 The gRPC Authors
```

Comment:
Why does this need to be checked in? It's versioned remotely so a copy might not be necessary, or is it?